data/vaults/0x5f18C75AbDAe578b483E5F43f12a39cF75b973a9.json

```diff
@@ -8,5 +8,6 @@
   "migrationAvailable": false,
   "allowZapIn": true,
   "allowZapOut": true,
-  "retired": false
+  "retired": false,
+  "displayName": ""
```

Comment:
since this property is optional maybe best not to have empty strings? we can just remove it I think

Reply:
lol good catch, I think I definitely just accidentally skipped this one

Reply:
fixed